index.js

```diff
@@ -12,6 +12,8 @@ const debug = require('debug')('couchbase-driver')
  * @memberof Driver
  */
 const OPERATIONS = {
+  /** Insert operation */
+  INSERT: 'insert',
```

Comment:
Yeah I wasn't 100% sure whether these were intended specifically for atomic or not, so I just added it there since it was another similar const. I'll clean that up tomorrow